src/Microsoft.ApplicationInsights.AspNet/ApplicationInsightsJavaScript.cs

```diff
@@ -1,19 +1,15 @@
 ﻿
-namespace Microsoft.ApplicationInsights.AspNet.JavaScript
+namespace Microsoft.ApplicationInsights.AspNet
 {
     using System;
     using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.AspNet.Mvc.Rendering;
+
 
     /// <summary>
     /// This class helps to inject Application Insights JavaScript snippet into applicaiton code.
-    /// Usage:
-    ///     Inject an instance of the class into your razor code using code:
-    ///         @inject Microsoft.ApplicationInsights.AspNet.JavaScript.IJavaScriptSnippet ApplicationInsightsJavaScript
-    ///     Place this code fragment just before closing "head" tag
-    ///         @Html.Raw(ApplicationInsightsJavaScript.Write())
-    ///     &lt;/head&gt;
     /// </summary>
-    public class ApplicationInsightsJavaScript : IJavaScriptSnippet
+    public class JavaScriptSnippet
     {
```

Comment:
So at this point, `JavaScriptSnippet` is an implementation detail of the `ApplicationInsightsJavaScript` extension method and does not need to be public. Correct? Is that something we care about?